src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const headers = [...document.querySelector('thead').children[0].children];
```

Comment:
Instead of using 'children[0].children', use 'querySelectorAll' to get all the 'th' elements.